lib/MDTable.js

```diff
@@ -0,0 +1,69 @@
+import AsciTable from 'ascii-table';
+
+class MDTable extends AsciTable {
+    /**
+     * Create a CSV string from the table data
+     * @returns {string}
+     */
+    toCSV() {
+        return [this.getHeading(), ...this.getRows()].join('\n');
+    }
+
+    /**
+     * Create a JSON string from the table data
+     * @returns {string}
+     */
+    toJSON() {
+        return JSON.stringify(super.toJSON());
+    }
+
+    /**
+     * Create a MD string from the table data
+     * @returns {string}
+     */
+    toString() {
+        // Ignore modifying the first |---| for titled tables
+        let idx = this.getTitle() ? -2 : -1;
+        const ascii = super.toString()
+            .replace(/-\|/g, () => {
+                /* we replace "----|" with "---:|" to align the data to the right in MD */
+                idx++;
+
+                if (idx < 0 || this.leftAlignedCols.includes(idx)) {
+                    return '-|';
+                }
+
+                return ':|';
+            });
```

Comment:
I guess this is the most weird part of the PR

Markdown tables are left aligned by default 
To align a column you add a `:` column on the side you want it to align (or on both sides for centered)
```
|----------|---------:|----------:|
```

When we're printing a table with a title we want to skip the first row which looks like:
```
|            Onyx:set            |
|--------------------------------|
```

That's why we start with `-2` on that case

Reply:
maybe could use a different token than `Onyx:set` like `Onyx~set` but this seems fine.

Reply:
Individual table names would be title with whatever alias is provided here: 
https://github.com/Expensify/react-native-onyx/blob/master/lib/Onyx.js#L771-L777